staging/src/k8s.io/component-base/metrics/registry.go

```diff
@@ -120,7 +121,7 @@ type KubeRegistry interface {
 	CustomMustRegister(cs ...StableCollector)
 	Register(Registerable) error
 	MustRegister(...Registerable)
-	Unregister(Registerable) bool
+	Unregister(collector Collector) bool
```

Comment:
I think we can share Unregister(collector Collector) between `normal metric`、custome collector and value functions.